### DISCUSSION TITLE
Dishwasher published no events

### DISCUSSION THREAD

**Discussion by nireins** [ekutner/home-connect-hass#304] [General]
Hello everyone, I'm desperate. My new dishwasher does not display any events via the HomeConnect alt integration.

In the HomeConncet app, however, it does.

What I've done so far:

In Home Assistant -> Developer Tools-> Events

Listen for events "home_connect_alt_event" and start listening.

The dishwasher currently shows “rinse aid empty” in the HC app as soon as I turn the dishwasher on.

However, this event is not displayed in Home Assistant.

Am I doing something wrong? Can someone tell me why that is? Many thanks and greetings

**Comment by kwbeck** [comment]
They are sent by the dishwasher, and exposed as Home Assistant Events by the integration,  but since they are "Events" not States, you have to set up something to listen to them.

There is a discussion here.

https://github.com/ekutner/home-connect-hass/discussions/81
